packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -18,25 +18,30 @@ import { repeat } from 'lit-html/directives/repeat.js';
 import { defaultTemplate } from './defaultTemplate';
 @customElement('opc-feedback')
 export class OpcFeedback extends LitElement {
-  @property({ type: String, attribute: 'url' }) url;
+  @property({ type: String, attribute: 'spaUrl' }) spaUrl;
+  @property({ type: String, attribute: 'docUrl' }) docUrl;
```

Comment:
```suggestion
  @property({ type: String, attribute: 'spa-url' }) spaUrl = '/feedback';
  @property({ type: String, attribute: 'doc-url' }) docUrl = '/get-started';
```
This will have default values which can be overridden with attributes

Reply:
Refer the constructor function;